packages/spacebars-tests/template_tests.js

```diff
@@ -3144,8 +3144,10 @@ testAsyncMulti("spacebars-tests - template_tests - template-level subscriptions"
 
     tmpl.onCreated(function () {
       var subHandle = this.subscribe("templateSub", subscribeCallback);
-      var subHandle2 = this.subscribe(
-        "templateSub", futureId, subscribeCallback);
+      var subHandle2 = this.subscribe("templateSub", futureId, {
+        onReady: subscribeCallback,
+        connection: Meteor.connection
```

Comment:
@stubailo I am sure this test does not cover a situation where the `connection` is different from the standard one.